File: python/7 kyu/21.py
# https://www.codewars.com/kata/595aa94353e43a8746000120/train/python

# Passed

def find_deleted_number(arr, mixed_arr):
    difference = set(arr) - set(mixed_arr)
    return difference.pop() if difference else 0

output = find_deleted_number([1, 2, 3, 4, 5], [1, 2, 4, 3])
print(output)

File: nodejs/7 kyu/21.js
// https://www.codewars.com/kata/5f0ed36164f2bc00283aed07/train/javascript

// Passed

function overTheRoad(address, n) {
  const maxHouses = 2 * n;
  const sumHouses = 1 + maxHouses;

  // sumHouses = oddHouse + evenHouse
  return sumHouses - address;
}


const output = overTheRoad(3, 5);
console.log(output);